lib/util/LabelUtil.js

```diff
@@ -251,7 +251,10 @@ export function setLabel(element, text) {
   if (attr) {
 
     if (attr === 'categoryValueRef') {
-      semantic['categoryValueRef'].value = text;
+      if (!semantic[attr]) {
+        return;
```

Comment:
Watch for the contract of this function. If you early return, I propose you `return element` to not break it.

Alternative: 

```
if (semantic[attr]) {
  // do change
}
```

My follow-up question is: Who calls this, and why, after the label has been removed? Will it ever be called with a non-empty text (in which case the above code would shadow a deeper issue?

Safe choice would be to be strict (ignore update only, if label is empty _and_ `categoryValueRef` is unset).

Reply:
I would agree to early return the `element`. 

The `updateLabel` event calls this method. Even if we remove the existing label, it is updated via this method. It is not called after the label is removed, but rather when we are updating the label during `execute` lifecycle.

Reply:
If the text is not empty, the `label.create` event will be fired and `setCategoryValue` function will be invoked in file `GroupBehavior.js`. So, in case the label is not empty, we will have `categoryValueRef`.